cmd2/cmd2.py

```diff
@@ -3113,7 +3117,7 @@ def quit():
             pass
         finally:
             self._in_py = False
-        return self._should_quit
+            return self._should_quit
```

Comment:
This interesting whitespace change seems innocuous.  But is there a good reason for it?